### PR TITLE
Ddcnls-6587: move to build.sbt and updated library dependencies

### PR DESCRIPTION
**Pull request by BriWak**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+val appName: String = "apprenticeship-levy"
+
+val XsltConfig = config("api-docs")
+val generateAPIDocs = TaskKey[Unit]("api-docs", "Generates HMRC API Documentation files")
+managedClasspath in generateAPIDocs := {
+  // these are the types of artifacts to include
+  val artifactTypes: Set[String] = (classpathTypes in generateAPIDocs).value
+  Classpaths.managedJars(XsltConfig, artifactTypes, update.value)
+}
+val generateAPIDocsTask = generateAPIDocs := {
```
Comment (daniel-manning) [inline build.sbt]: have you run these and seen it working?

File: build.sbt [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    playSettings,
+    scoverageSettings,
+    scalaSettings,
+    publishingSettings,
+    defaultSettings(),
+    PlayKeys.playDefaultPort := 9470,
+    majorVersion := 3,
+    ivyConfigurations += XsltConfig,
+    libraryDependencies ++= AppDependencies.all,
+    libraryDependencies += "net.sourceforge.saxon" % "saxon" % "9.1.0.8" % XsltConfig.name,
```
Comment (daniel-manning) [inline build.sbt]: move this in to App dependencies but call it generateApiTask, keep it separate from the all variable

File: build.sbt (hunk 1 above)
Comment (BriWak) [reply build.sbt]: Yes, it is working as it was in master

File: build.sbt (hunk 2 above)
Comment (BriWak) [reply build.sbt]: Done

**Pull request merged**